docs/troubleshooting.md

```diff
@@ -32,6 +34,11 @@ kubectl edit deployment/ark -n heptio-ark
 ...
 ```
 
+## Known issue with restoring LoadBalancer Service
+
+Because of how Kubernetes handles Service objects of `type=LoadBalancer`, you might encounter an issue with changed values for Service UIDs. If the DNS CNAME for your application points to the DNS name of your cloud load balancer, you'll need to update the CNAME pointer when you perform an Ark restore.
+
+Alternatively, if your cloud provider supports an external load balancer, you might be able to use the Service's `spec.loadBalancerIP` field to keep connections valid. See [the Kubernetes documentation about Services of Type LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). For more information, see [the documentation about how to create an External Load Balancer while preserving the client source IP](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip).
```

Comment:
"Alternatively, you might be able to use the Service's `spec.loadBalancerIP` field if your cloud provider supports it to keep connections valid."

Also, that last sentence/link is not relevant here so can be removed.